### PR TITLE
Client side merge

### PR DESCRIPTION
**Pull request by xori**
Having this project split between two projects has been very unmotivating to work on. Now they're merged together.

To build:
1. build the java backend first
1. `npm start` inside the `frontend` submodule directory

### CODE REVIEW

File: src/main/java/com/ace/alfox/AlfoxApplication.java [hunk 1]
```diff
@@ -5,11 +5,10 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-@EnableWebMvc
 @SpringBootApplication
-@EnableAutoConfiguration
```
Comment (xori) [inline src/main/java/com/ace/alfox/AlfoxApplication.java:12]: This was hell to find. Apparently you can't have both `@EnableWebMvc` and `@SpringBootApplication` or it clears out the default configuration and won't serve any static files.

Comment (coltin) [reply src/main/java/com/ace/alfox/AlfoxApplication.java:12]: Yeahhhhhh

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+server.error.whitelabel.enabled=false
```
Comment (coltin) [inline src/main/resources/application.properties]: What does this do?

**Review by coltin — APPROVED** [review]

File: src/main/resources/application.properties (hunk 2 above)
Comment (xori) [reply src/main/resources/application.properties]: it hides this


but that's better than tomcat


so I'm going to revert this change.

**Pull request merged**